ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -1628,7 +1628,10 @@ def getFundamentalFeatures(self, requestedFeatures, featureTemplate=None):
           ## IND
           #most probabble index
           if len(group['Ind']):
-            modeInd = stats.mode(group['Ind'])[0][0]
+            try:
```

Comment:
Could you provide a comment here to explain why we need the try except here?

Reply:
Hm, depending on version of Scipy, we get a different shaped array because of the change of the keepdims in stats.mode https://docs.scipy.org/doc/scipy/release/1.11.0-notes.html#expired-deprecations

So probably we should just specify keepdims here and use only one of these options.

Reply:
I switched it to use an explicit keepdims.